### ISSUE TITLE
When IPY.EXE starts, IPY.EXE crashes.

### ISSUE THREAD

**Issue by ishisaka** [IronLanguages/main#1346]
When IPY.EXE starts, IPY.EXE crashes.

IPY.EXE Version 2.7.6.3
Windows 10 Version 1607(Build 14393.51)
.NET Frame Work Version 4.6.2

Exception:
```
> "C:\Program Files (x86)\IronPython 2.7\ipy.exe"

ハンドルされていない例外: System.Configuration.ConfigurationErrorsException: microsoft.scripting のための構成セクション ハンドラーを
作成中にエラーが発生しました。: ファイルまたはアセンブリ 'Microsoft.Scripting, Version=1.1.0.20, Culture=neutral, PublicKeyToken=7f7
09c5b713576e1'、またはその依存関係の 1 つが読み込めませんでした。見つかったアセンブリのマニフェスト定義はアセンブリ参照に一致しませ
ん。 (HRESULT からの例外:0x80131040) (C:\Program Files (x86)\IronPython 2.7\ipy.exe.Config line 5) ---> System.IO.FileLoadException:
 ファイルまたはアセンブリ 'Microsoft.Scripting, Version=1.1.0.20, Culture=neutral, PublicKeyToken=7f709c5b713576e1'、またはその依存
関係の 1 つが読み込めませんでした。見つかったアセンブリのマニフェスト定義はアセンブリ参照に一致しません。 (HRESULT からの例外:0x8013
1040)
   場所 System.Configuration.TypeUtil.GetTypeWithReflectionPermission(IInternalConfigHost host, String typeString, Boolean throwOnEr
ror)
   場所 System.Configuration.RuntimeConfigurationRecord.RuntimeConfigurationFactory.Init(RuntimeConfigurationRecord configRecord, Fa
ctoryRecord factoryRecord)
   場所 System.Configuration.RuntimeConfigurationRecord.RuntimeConfigurationFactory.InitWithRestrictedPermissions(RuntimeConfigurati
onRecord configRecord, FactoryRecord factoryRecord)
   場所 System.Configuration.RuntimeConfigurationRecord.CreateSectionFactory(FactoryRecord factoryRecord)
   場所 System.Configuration.BaseConfigurationRecord.FindAndEnsureFactoryRecord(String configKey, Boolean& isRootDeclaredHere)
   --- 内部例外スタック トレースの終わり ---
   場所 System.Configuration.BaseConfigurationRecord.FindAndEnsureFactoryRecord(String configKey, Boolean& isRootDeclaredHere)
   場所 System.Configuration.BaseConfigurationRecord.GetSectionRecursive(String configKey, Boolean getLkg, Boolean checkPermission,
Boolean getRuntimeObject, Boolean requestIsHere, Object& result, Object& resultRuntimeObject)
   場所 System.Configuration.BaseConfigurationRecord.GetSection(String configKey)
   場所 System.Configuration.ClientConfigurationSystem.System.Configuration.Internal.IInternalConfigSystem.GetSection(String section
Name)
   場所 System.Configuration.ConfigurationManager.GetSection(String sectionName)
   場所 Microsoft.Scripting.Hosting.Configuration.Section.LoadRuntimeSetup(ScriptRuntimeSetup setup, Stream configFileStream)
   場所 PythonConsoleHost.CreateRuntimeSetup()
   場所 Microsoft.Scripting.Hosting.Shell.ConsoleHost.Run(String[] args)
   場所 PythonConsoleHost.Main(String[] args)
```

**Comment by andygraham** [comment]
I commented on this on Gitter. ishisaka beat me to logging it as an issue!

I'm on Windows 764bit Home with .NET 4.6.2.

> 
Unhandled Exception: System.Configuration.ConfigurationErrorsException: An error
 occurred creating the configuration section handler for microsoft.scripting: Co
uld not load file or assembly 'Microsoft.Scripting, Version=1.1.0.20, Culture=ne
utral, PublicKeyToken=7f709c5b713576e1' or one of its dependencies. The located
assembly's manifest definition does not match the assembly reference. (Exception
 from HRESULT: 0x80131040) (C:\IronPython 2.7\ipy.exe.Config line 5) ---> System
.IO.FileLoadException: Could not load file or assembly 'Microsoft.Scripting, Ver
sion=1.1.0.20, Culture=neutral, PublicKeyToken=7f709c5b713576e1' or one of its d
ependencies. The located assembly's manifest definition does not match the assem
bly reference. (Exception from HRESULT: 0x80131040)
   at System.Configuration.TypeUtil.GetTypeWithReflectionPermission(IInternalCon
figHost host, String typeString, Boolean throwOnError)
   at System.Configuration.RuntimeConfigurationRecord.RuntimeConfigurationFactor
y.Init(RuntimeConfigurationRecord configRecord, FactoryRecord factoryRecord)
   at System.Configuration.RuntimeConfigurationRecord.RuntimeConfigurationFactor
y.InitWithRestrictedPermissions(RuntimeConfigurationRecord configRecord, Factory
Record factoryRecord)
   at System.Configuration.RuntimeConfigurationRecord.CreateSectionFactory(Facto
ryRecord factoryRecord)
   at System.Configuration.BaseConfigurationRecord.FindAndEnsureFactoryRecord(St
ring configKey, Boolean& isRootDeclaredHere)
   --- End of inner exception stack trace ---
   at System.Configuration.BaseConfigurationRecord.FindAndEnsureFactoryRecord(St
ring configKey, Boolean& isRootDeclaredHere)
   at System.Configuration.BaseConfigurationRecord.GetSectionRecursive(String co
nfigKey, Boolean getLkg, Boolean checkPermission, Boolean getRuntimeObject, Bool
ean requestIsHere, Object& result, Object& resultRuntimeObject)
   at System.Configuration.BaseConfigurationRecord.GetSection(String configKey)
   at System.Configuration.ClientConfigurationSystem.System.Configuration.Intern
al.IInternalConfigSystem.GetSection(String sectionName)
   at System.Configuration.ConfigurationManager.GetSection(String sectionName)
   at Microsoft.Scripting.Hosting.Configuration.Section.LoadRuntimeSetup(ScriptR
untimeSetup setup, Stream configFileStream)
   at PythonConsoleHost.CreateRuntimeSetup()
   at Microsoft.Scripting.Hosting.Shell.ConsoleHost.Run(String[] args)
   at PythonConsoleHost.Main(String[] args)

>

**Comment by slozier** [comment]
Look like the config file are still pointing to `Microsoft.Scripting, Version=1.1.0.20` hence the crash. As a workaround until the release if fixed you can edit the ipy.exe.config file and replace `1.1.0.20` with `1.1.1.21`.

**Comment by slide** [comment]
Fixed in 4cb8abc
